module/VuFind/src/VuFind/Db/Table/PluginManager.php

```diff
@@ -49,7 +49,6 @@ class PluginManager extends \VuFind\ServiceManager\AbstractPluginManager
         'comments' => Comments::class,
         'externalsession' => ExternalSession::class,
         'oairesumption' => OaiResumption::class,
-        'ratings' => Ratings::class,
```

Comment:
Is there also a Db/Row class that should be removed as well?

Reply:
Storage of Ratings is still Running on Laminas as it has many dependencies. So this has to be persisted. Will remove it once it was moved in an another pull request.

Reply:
Makes sense -- but do you mind removing the methods from the row class that have been refactored to the service, so we can more easily keep track of which work is done and which work is not?